packages/context/src/binding-decorator.ts

```diff
@@ -110,3 +113,18 @@ export namespace bind {
     };
   }
 }
+
+/**
+ * Alias decorator `@injectable` to `@injectable` for backward compatibility
+ * @param specs Binding specs
+ */
+export function bind(...specs: BindingSpec[]): ClassDecorator {
+  return injectable(...specs);
+}
+
+/**
+ * Alias namespace `bind` to `injectable` for backward compatibility
```

Comment:
Should we add `@deprecated` keyword and point users to the new `@injectable` decorator?